exercises/hello-world/hello_test.go

```diff
@@ -2,25 +2,17 @@ package greeting
 
 import "testing"
 
-// Define a function HelloWorld(string) string.
+// Define a function HelloWorld() string.
```

Comment:
The way this reads is a little confusing because of the return type of the function signature, it takes a second to understand that the word string at the end is part of the function, rather than it saying *Define a function string*.
Maybe moving the function signature to the next line will help?
```
// Define a function with the following signature/the following function (??)
// HelloWorld() string